pinery-ws/src/main/java/ca/on/oicr/pinery/ws/LaneProvenanceResource.java

```diff
@@ -61,8 +68,8 @@
       = new MapBuilder<String, VersionTransformer<LaneProvenance, ? extends LaneProvenance>>() //
           .put("latest", noopTransformer) //
           .put("v5", noopTransformer) //
-          .put("v4", noopTransformer) //
-          .put("v3", noopTransformer) //
+          .put("v4", v3Transformer) //
```

Comment:
It's a bit confusing to run the v3Transformer when v4 is selected; why not add a v4 transformer?

Reply:
v4 lane provenance had no changes, so it's the same as v3